docs/library/pyb.SPI.rst

```diff
@@ -66,6 +66,7 @@ Methods
            use of ``prescaler`` overrides ``baudrate``.
          - ``polarity`` can be 0 or 1, and is the level the idle clock line sits at.
          - ``phase`` can be 0 or 1 to sample data on the first or second clock edge
+         - ``bits`` can be 8 or 16, and is the data size.
```

Comment:
I would find "and is the data size" part a bit confusing, why I don't merge this and leave for further review/discussion.